File: 3960-number-of-student-replacements/3960-number-of-student-replacements.py
class Solution:
    def totalReplacements(self, ranks: List[int]) -> int:
        if len(ranks)==0:
            return 0
        minRank=ranks[0]
        countReplacement=0

        for i in range(1,len(ranks)):
            if ranks[i]<minRank:
                minRank=ranks[i]
                countReplacement+=1
        return countReplacement
        

File: 3960-number-of-student-replacements/3960-number-of-student-replacements.ts
function totalReplacements(ranks: number[]): number {
     if(ranks.length==0){
        return 0
     }
     let minRank:number=ranks[0]
     let countReplacement=0

     for(let i=1;i<ranks.length;i++){
        if(minRank>ranks[i]){
            minRank=ranks[i]
            countReplacement++
        }
     }
     return countReplacement
    
};